### ISSUE TITLE
Add alert role where there's an alert to improve accessibility.

### ISSUE THREAD

**Issue by renujain31** [ushahidi/platform#4109]
# Description:
Alerts are used to convey messages to alert the user. In the case of audio warnings this is an accessible alternative for a hearing-impaired user. We can add alerts by using `role=alert` in the HTML element and thus can increase the accessibility of platform.

# Todo:
Search for `class="alert"` or `class="alert error"` in the codebase of platform-client and add `role=alert` in the html element with that class. Make sure that you do not make any changes to the UI of the platform.

**Comment by stacytonui** [comment]
Hello, please assign this to me, thanks!

**Comment by rtwitty72** [comment]
Hi @renujain31, I would like to resolve this issue, please?

**Comment by renujain31** [comment]
Hey. @Angamanga Can you please assign this issue to @stacytonui I am unable to assign as I don't have permission for it. Thanks!

@rtwitty72 I would create another issue for you.

**Comment by Dev-Som** [comment]
@renujain31 Can I please get an issue to work on?

**Comment by Angamanga** [comment]
@stacytonui I have assigned you to this issue. @rtwitty72 @Dev-Som We are working on adding more issues continuously, check the community-task label as new issues will appear there!

**Comment by emmanuelhashy** [comment]
Hi @Angamanga I'd like to work on this issue. Assign it to me please.

**Comment by Angamanga** [comment]
@emmanuelhashy Yes, please go ahead. I have reviewed your PR so make sure to finish that first 👍

**Comment by JUNNY12** [comment]
Can I work on this issue? I am from Outreachy Internship

**Comment by Angamanga** [comment]
@JUNNY12 Welcome to Ushahidi! In order to start an issue, you need to follow the steps in this discussion: https://github.com/ushahidi/platform/discussions/4713 Have you set up your developer environment yet?

**Comment by JUNNY12** [comment]
Thank you @Angamanga . Yes I have done that. Below is a screenshot.
![setup](https://github.com/ushahidi/platform/assets/81489510/9a62796b-1e12-4b17-a297-0109c3f76994)

**Comment by Angamanga** [comment]
Thank you @JUNNY12! Please go ahead working on this issue :)

**Comment by JUNNY12** [comment]
Hello @Angamanga , please this is a kind reminder to review the pull request on the issue you assigned to me. Here's to he link to the pull request; [Link](https://github.com/ushahidi/platform-client-mzima/pull/807)

**Comment by Angamanga** [comment]
Closing this issue since outdated in new platform-client
